avm/res/network/front-door/tests/e2e/max/main.test.bicep

```diff
@@ -193,12 +192,6 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
         storageAccountResourceId: diagnosticDependencies.outputs.storageAccountResourceId
         workspaceResourceId: diagnosticDependencies.outputs.logAnalyticsWorkspaceResourceId
       }
-      {
```

Comment:
intended?

Reply:
Yes, fixes test (see comment to your orginal PE test comment)